nasa-gibs/worldview
# WV-3289: Add OMPS NOAA-20 SO2 and Suomi NPP LP Aerosol layers

## Description

Fixes #WV-3289

Add 11 layers - 4 NOAA-20/OMPS SO2 layers, 7 Suomi NPP/OMPS LP layers

NOAA-20/OMPS
1. Sulfur Dioxide (Planetary Boundary Layer)
2. Sulfur Dioxide (Upper Troposphere and Stratosphere)
3. Sulfur Dioxide (Lower Troposphere)
4. Sulfur Dioxide (Middle Troposphere)

Suomi NPP/OMPS LP
1. Aerosol Extinction Vertical Profile (Aerosol height)
2. Aerosol Extinction Vertical Profile (11.5 - 12.5 km)
3. Aerosol Extinction Vertical Profile (13.5 - 14.5 km)
4. Aerosol Extinction Vertical Profile (15.5 - 16.5 km)
5. Aerosol Extinction Vertical Profile (17.5 - 18.5 km)
6. Aerosol Extinction Vertical Profile (19.5 - 20.5 km)
7. Stratospheric Aerosol Optical Depth

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,OMPS_NOAA20_SO2_Upper_Troposphere_and_Stratosphere,OMPS_NOAA20_SO2_Middle_Troposphere,OMPS_NOAA20_SO2_Lower_Troposphere,OMPS_NOAA20_SO2_Planetary_Boundary_Layer,OMPS_SNPP_LimbProfiler_Aerosol_ExinctionCoefficient_20KM,OMPS_SNPP_LimbProfiler_Aerosol_ExinctionCoefficient_18KM,OMPS_SNPP_LimbProfiler_Aerosol_ExinctionCoefficient_16KM,OMPS_SNPP_LimbProfiler_Aerosol_ExinctionCoefficient_14KM,OMPS_SNPP_LimbProfiler_Aerosol_ExinctionCoefficient_12KM,OMPS_SNPP_LimbProfiler_AerosolHeight,OMPS_SNPP_LimbProfiler_Aerosol_OpticalDepth,VIIRS_NOAA21_CorrectedReflectance_TrueColor(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-11-07-T16%3A56%3A53Z)
2. Ensure that layers load and show imagery, titles and layer descriptions are readable, you can take an image snapshot of the layers. 
3. Also check they are categorised correctly: Go to Add Layers, Select "Aerosol Extinction", "Aerosol Optical Depth" for Suomi NPP layers; "Sulfur Dioxide" for NOAA-20 layers.
4. Verify expected result


## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
****
